src/test/java/com/github/couchmove/CouchmoveIT.java

```diff
@@ -53,9 +53,9 @@ public class CouchmoveIT extends BaseIT {
 
     @BeforeEach
     public void init() {
-        changeLogRepository = new CouchbaseRepositoryImpl<>(getCluster(), getBucket(), ChangeLog.class);
-        changeLogDBService = new ChangeLogDBService(getBucket(), getCluster());
-        userRepository = new CouchbaseRepositoryImpl<>(getCluster(), getBucket(), User.class);
+        changeLogRepository = new CouchbaseRepositoryImpl<>(getCluster(), getBucket(), ChangeLog.class, Collections.emptyMap());
```

Comment:
This change can be avoided by adding new constructor without variables

Reply:
Done